weather.py

```diff
@@ -248,7 +249,7 @@ def __init__(self):
         # for fontname in pygame.font.get_fonts():
         #        print(fontname)
 
-        if config.FULLSCREEN:
+        if CONFIG["fullscreen"] == "yes":
```

Comment:
why switch away from the boolean here?

Reply:
I'm still looking into ways of saving integers and booleans via javascript. JS is new to me. This is one of a couple things that I need to amend yet. I wanted to get the PR in so that I could start getting feedback.

Reply:
I guess it's an HTML limitation, but I think I can find a workaround. It will help make the main python modules more efficient, too (As I had to change some conditionals from `if xyz` to `if xyz == "yes"`, which is unnecessarily verbose)

https://stackoverflow.com/questions/53745774/get-boolean-option-values

Reply:
took me a while, but I have the config file saving int, bool and strings properly now.